tests/README.md

```diff
@@ -77,17 +77,18 @@ The build script has the following parameters:
 Select `Ubuntu 14.04 LTS v1503 (beta with Docker support)`.
 The platform with *Docker support* means the tests will run in a VM.
 
-## Manually running the functional tests
+## Manually running the tests
 
 Make sure to pass `--enable-functional-tests` to the configure script, then, after building the project, you can run the tests.
```

Comment:
Without `--enable-functional-tests`, unit tests work, so it should still say "functional tests".